spdx-utils/src/main/kotlin/SpdxExpression.kt

```diff
@@ -491,6 +533,20 @@ data class SpdxLicenseReferenceExpression(
     override fun getLicenseUrl(): String? = null
 }
 
+/**
+ * Aggregates multiple SpdxExpressions to a [SpdxCompoundExpression] concatenated using [SpdxOperator.OR].
+ */
+fun List<SpdxExpression>.choicesToSpdx(): SpdxExpression {
+    if (size == 1) return this[0]
```

Comment:
`first()` instead of `this[0]`? Same below.

Reply:
Not relevant anymore